app/controllers/v1/profiles_controller.rb

```diff
@@ -74,7 +75,7 @@ def tailoring_filename
     end
 
     def profile
-      @profile ||= pundit_scope.find(params[:id])
+      @profile ||= @new_profile || pundit_scope.find(params[:id])
```

Comment:
This is a little misleading, I guess. `@new_profile` will be `nil` for updates. Only calling the `new_profile` method will set this variable, but once it's set, we want to be sure we use it instead of finding a profile. I initially wrote it so that `new_profile` memoizes `@profile`, but rubocop complains when the memoized variable doesn't match the method name. :shrug:

Reply:
Oh right, the instance variable would be empty. :+1: